assets/js/googlesitekit/datastore/user/key-metrics.test.js

```diff
@@ -415,10 +453,7 @@ describe( 'core/user key metrics', () => {
 
 					expect(
 						registry.select( CORE_USER ).getAnswerBasedMetrics()
-					).toEqual( [
-						...expectedMetrics,
-						...conversionTailoredMetrics,
-					] );
+					).toEqual( conversionTailoredMetrics );
```

Comment:
Hmm, if I've understood things correctly the new set of metrics is a mix of the old/non-conversion and conversion based metrics, so it would be good to rename this variable e.g. `expectedMetricsIncludingConversionTailored` or something along those lines...

```suggestion
					).toEqual( expectedMetricsIncludingConversionTailored );
```